media-sound/picard/picard-2.1.3.recipe

```diff
@@ -33,6 +33,7 @@ REQUIRES="
 	pyqt_python3
 	mutagen_python36
 	discid_python36
+	cmd:fpcalc
```

Comment:
Please sort this section alphabetically.

Reply:
Ok, did this, but kept the haiku dependency at the top because this is what I usually see in other recipes. Is this how it is expected? Not sure if I like this the best currently. What about some rule like:

1. First list haiku[_devel]
2. Then list dependencies with prefixes (`cmd:`, `lib:` and probably others?) alphabetically
3. List all other dependencies alphabetically